packages/integration-karma/test/light-dom/slotting/index.spec.js

```diff
@@ -15,11 +16,19 @@ function createTestElement(tag, component) {
     return extractDataIds(elm);
 }
 
+function removeSlotMarker(html) {
+    return html.replace(/\u200D/gi, '');
+}
+
+function filterOutSlotMarker(node) {
+    return !(node.nodeType === Node.TEXT_NODE && node.nodeValue === '\u200D');
```

Comment:
Can we be confident that this won't conflict with user code?

Reply:
i don't think so, what is creating a vnode for the marker (`\u200D`), which is going to be different than the one from the user code. but now that you mention it 🤔 , maybe I can generate an empty vnode, that will definitely be better.

Reply:
ja! great comment @nolanlawson. 😅 turns out an empty text marker does the trick.